src/Stryker.Core/Stryker.Core/Mutants/MutantOrchestrator.cs

```diff
@@ -31,6 +32,7 @@ public class MutantOrchestrator : IMutantOrchestrator
         private IEnumerable<IMutator> _mutators { get; set; }
         private ILogger _logger { get; set; }
 
+        private static Type[] expressionStatementNeedingIf = { typeof(AssignmentExpressionSyntax), typeof(PostfixUnaryExpressionSyntax), typeof(PrefixUnaryExpressionSyntax)};
```

Comment:
Why is this static?

Reply:
It is supposed to be a constant. I marked it as readonly. Why do you think it should be an instance field?

Reply:
Ah, I wanted it to be `const` but of course that doesn't work. We could add the readonly keyword?

Reply:
Now I see you already did, resolved!